_topic_map.yml

```diff
@@ -1436,6 +1436,8 @@ Topics:
   Topics:
   - Name: Traffic management
     File: ossm-traffic-manage
+  - Name: Data visualization and observability
```

Comment:
Kiali is not only for observability:
https://kiali.io/

Reply:
I hear you. This is the start. I think this is the right title for the items in the PR right now. As we add content and features to this topic, I can adjust the title of the topic then. 

When I titled the PR intro, I think I meant for it to be a start on the documentation that we need here. It's only a couple of items on a bigger epic. I also think the features and architechture are discussed broadly here: 
 
https://docs.openshift.com/container-platform/4.3/service_mesh/service_mesh_arch/ossm-kiali.html